cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -16,22 +16,6 @@ spec:
     spec:
       serviceAccount: rook-csi-cephfs-provisioner-sa
       containers:
-        - name: csi-attacher
```

Comment:
Should this be left but commented out with a note that it is required for k8s 1.13.x and below?

Reply:
@Madhu-1 Can you clarify what is needed for the attacher? It's only needed for v1.13? In that case we would still need to create this for 1.13, right?

Reply:
Is removing the attacher related to registering the csi driver? Seems like it should be a separate PR?

Reply:
@BlaineEXE  if kube is less than 1.14 we are deploying csi as statefulset https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-sts.yaml, this is not required

@travisn attacher is removed by creating the CSIDriver attacher=false as mentioned here https://kubernetes-csi.github.io/docs/skip-attach.html#skip-attach-with-csi-driver-object

Reply:
Ok, so we don't need to leave this for 1.13 because we're using the statefulset in that case.